### DISCUSSION TITLE
Not selecting first frame in video?

### DISCUSSION THREAD

**Discussion by JakeHolcroft** [ThatGuySam/vimeo-thumbnail-server#41] [Q&A] [answered]

**Comment by JakeHolcroft** [comment]
solved

**Comment by ThatGuySam** [comment]
@JakeHolcroft Fantastic! Care to share the solution for future internet travelers?

**Comment by JakeHolcroft** [comment, marked as answer]
Yeah, set the frame you want as your thumbnail within Vimeo :P I misunderstood how this library worked, i thought it just grabbed the first frame of the video as a thumbnail but it grabs your selected thumbnail.

Thanks

**Reply by ThatGuySam** [reply]
@JakeHolcroft Beautiful! Feel to reach out if there's anything else
